### PR TITLE
Adds Spurdo Hardsuits

### PR DESCRIPTION
**Pull request by GuppyLaxx**
Adds spurdo suits, replaces your screams with pyko swearing too, admin only and has combat stats for now, idk about the stats or how to get it, so suggest it up, biggiOS made the sprite.

### CODE REVIEW

File: hippiestation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -22,3 +22,32 @@
...
+
+/obj/item/clothing/head/helmet/space/hardsuit/spurdosuit
+	name = "emergency spurdo suit helmet"
+	desc = ":-DDD"
+	icon_state = "spurdo"
+	item_state = "spurdo"
+	icon = 'hippiestation/icons/obj/clothing/masks.dmi'
+	alternate_worn_icon = 'hippiestation/icons/mob/head.dmi'
+	alternate_screams = list('hippiestation/sound/voice/finnish_1.ogg', 'hippiestation/sound/voice/finnish_2.ogg', 'hippiestation/sound/voice/finnish_3.ogg')
+	armor = list("melee" = 65, "bullet" = 50, "laser" = 50, "energy" = 50, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 80)
```
Comment (yoyobatty) [inline hippiestation/code/modules/clothing/spacesuits/hardsuit.dm]: This is better than almost every single combat hardsuit.

**Comment by yoyobatty** [comment]
I would use spacesuit armor stats instead, this is way too high stats

**Comment by GuppyLaxx** [comment]
these are just copy pasted stats from another suit, as i said, shoot me a suggestion for the stats, there's no way to get the suit rn

**Comment by yoyobatty** [comment]
`armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 50, "fire" = 80, "acid" = 70)`

**Comment by biggiOS** [comment]
stats as above ,basically a good space suit also add it to hacked clowndrobe kthx

**Comment by biggiOS** [comment]
hacked not emagged to be exact

**Comment by Carbonhell** [comment]
+1 but don't add it to the clowndrobe, make it adminspawn only,i cannot think of any place where this would suit

**Comment by biggiOS** [comment]
how about making it craftable by getting an eva suit, eva helmet,  10 sheets of plasteel, a brown crayon and a screwdriver

**Comment by nalarmg** [comment]
Code is fine.

**Comment by ghost** [comment]
seems really stupid...aye make it admin spawnable only

+1

**Comment by Carbonhell** [comment]
Requires changelog. +1

File: hippiestation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -22,3 +22,32 @@
...
+
+/obj/item/clothing/suit/space/hardsuit/spurdosuit
+	name = "emergency spurdo suit"
+	desc = "For those situations when you lack finnish people and combat gear."
+	icon = 'hippiestation/icons/obj/clothing/suits.dmi'
+	alternate_worn_icon = 'hippiestation/icons/mob/suit.dmi'
+	icon_state = "spurdo_suit"
+	item_state = "spurdo_suit"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/spurdosuit
+	allowed = list(/obj/item/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/tank/internals)
```
Comment (Carbonhell) [inline hippiestation/code/modules/clothing/spacesuits/hardsuit.dm:50]: kinda weird for a meme suit

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
Merging since it is admin spawn only

**Pull request merged**